oas_docs/output/kibana.yaml

```
@@ -359,7 +359,12 @@ tags:
       url: https://www.elastic.co/docs/deploy-manage/distributed-architecture/kibana-tasks-management
     name: task manager
     x-displayName: Task manager
-  - description: The assistant helps you prepare for the next major version of Elasticsearch.
+  - description: |
+      The Upgrade Assistant helps you prepare for the next major Elasticsearch release via the Kibana REST API.
```

```suggestion
      The Kibana Upgrade Assistant API helps you prepare for the next major Elasticsearch release.
```

Sorry I meant to include this change in first review but forgot to hit send 

Also I think we can just revert the moon.yml changes which are irrelevant here 🌚